### PR TITLE
Better test

### PR DESCRIPTION
**Pull request by P-Shakibafar**

### CODE REVIEW

**Comment by imanghafoori1** [comment]
resolve conflicts

**Comment by P-Shakibafar** [comment]
resolved conflicts please review again.

File: tests/MethodsClassTest.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Imanghafoori\LaravelMicroscope\Tests;
+
+use function file_get_contents;
```
Comment (imanghafoori1) [inline tests/MethodsClassTest.php:5]: do not use functions like this

**Comment by imanghafoori1** [comment]
please watch the latest clip on the telegram channel about microscope tests

**Comment by imanghafoori1** [comment]
always use assertEquals and avoid assertTrue and etc

**Comment by P-Shakibafar** [comment]
I watched, I think we can use setUp for duplicate code in tests and it is useful.

**Comment by P-Shakibafar** [comment]
Why use assertEquals instead of other methods? I did not catch! All of these methods use assertThat behind the scenes.

**Comment by imanghafoori1** [comment]
because it reads better. just that

**Review by P-Shakibafar — COMMENTED** [review]
please check conflict.

**Comment by imanghafoori1** [comment]
I do not want to use @dataProvider feature of phpunit, lets keep the tests as simple as possible.

**Comment by imanghafoori1** [comment]
Thanks for the suggestions. I applied them in my own commits.
Keep in mind you better create a single PR for each change. so that I can decide on each separately.

**Pull request closed without merging**